localstack-core/localstack/aws/protocol/service_router.py

```diff
@@ -192,8 +200,9 @@ def legacy_rules(request: Request) -> Optional[ServiceModelIdentifier]:
 
     # TODO Remove once fallback to S3 is disabled (after S3 ASF and Cors rework)
     # necessary for correct handling of cors for internal endpoints
-    if path.startswith("/_localstack") or path.startswith("/_pods") or path.startswith("/_aws"):
-        return None
+    for prefix in well_known_path_prefixes:
+        if path.startswith(prefix):
+            return None
```

Comment:
tiny nit:
```suggestion
    if any(path.startswith(prefix) for prefix in well_known_path_prefixes):
        return None
```

Reply:
i suppose this is highly subjective, but i don't perceive this as more readable. maybe it takes some getting used to :-)

Reply:
Agreed yes, I've learned to really like `all` and `any`, but doing a quick test, seems like it's actually slower 😄 forget the nit 😄

Reply:
always happy to have debates over style ;-)